pyproject.toml

```
@@ -1,28 +1,91 @@
 [tool.poetry]
 name = "hack2022-dapla-hurtigstart"
-version = "1.0.0"
-description = ""
+version = "0.0.0"
+description = "Hack2022 Dapla Hurtigstart"
 authors = ["Arne Sørli <81353974+arneso-ssb@users.noreply.github.com>"]
 license = "MIT"
 readme = "README.md"
-packages = [{include = "hurtigstart"}, {include = "create_github_repo"}]
+homepage = "https://github.com/statisticsnorway/hack2022-dapla-hurtigstart"
+repository = "https://github.com/statisticsnorway/hack2022-dapla-hurtigstart"
+documentation = "https://hack2022-dapla-hurtigstart.readthedocs.io"
+packages = [
+    { include = "dapla_hurtigstart_cli", from = "src" },
+]
+classifiers = [
+    "Development Status :: 1 - Planning",
+]
+
+[tool.poetry.urls]
+Changelog = "https://github.com/statisticsnorway/hack2022-dapla-hurtigstart/releases"
 
 [tool.poetry.dependencies]
-python = "^3.10"
+python = "^3.9"
+click = ">=8.0.1"
+dparse = "^0.6.2"
+typer = "^0.6.1"
 GitPython = "^3.1.27"
 PyGithub = "^1.55"
-typer = "^0.6.1"
 toml = "^0.10.2"
 cruft = "^2.11.1"
 rich = "^12.5.1"
+urllib3 = "^1.26.12"
+requests = "^2.28.1"
 
-[tool.poetry.group.dev.dependencies]
-black = "^22.8.0"
-isort = "^5.10.1"
+[tool.poetry.dev-dependencies]
+Pygments = ">=2.10.0"
+black = ">=21.10b0"
+coverage = {extras = ["toml"], version = ">=6.2"}
+darglint = ">=1.8.1"
+flake8 = ">=4.0.1"
+flake8-bandit = ">=2.1.2"
+flake8-bugbear = ">=21.9.2"
+flake8-docstrings = ">=1.6.0"
+flake8-rst-docstrings = ">=0.2.5"
+furo = ">=2021.11.12"
+isort = ">=5.10.1"
+mypy = ">=0.930"
+pep8-naming = ">=0.12.1"
+pre-commit = ">=2.16.0"
+pre-commit-hooks = ">=4.1.0"
+pytest = ">=6.2.5"
+pyupgrade = ">=2.29.1"
+safety = ">=1.10.3"
+sphinx = ">=4.3.2"
+sphinx-autobuild = ">=2021.3.14"
+sphinx-click = ">=3.0.2"
+typeguard = ">=2.13.3"
+xdoctest = {extras = ["colors"], version = ">=0.15.10"}
+myst-parser = {version = ">=0.16.1"}
+mock = "^4.0.3"
 
 [tool.poetry.scripts]
-ssb-project = 'hurtigstart.hurtigstart_cli:main'
+hurtigstart = "dapla_hurtigstart_cli.__main__:main"
```

Let's keep this as it was

```suggestion
ssb-project = "dapla_hurtigstart_cli.__main__:main"
```